src/menu.js

```diff
@@ -0,0 +1,54 @@
+import listItem from './listItem';
+
+const menu = () => {
+  const div = document.createElement('div');
+  div.setAttribute('id', 'main-content');
+  const menuContainer = document.createElement('div');
+  menuContainer.classList.add('menu-container');
+  const h2 = document.createElement('h2');
+  h2.textContent = 'Gallery';
+  const ul = document.createElement('ul');
+  ul.setAttribute('class', 'dish-list');
+  let img = document.createElement('img');
+  img.src = '../dist/assets/images/maki-sushi.jpg';
+  img.alt = 'Maki sushi';
+  let li = listItem('dish', 'maki-sushi', 'Maki sushi');
+  li.childNodes[0].classList.add('dish-description');
+  li.appendChild(img);
+  ul.appendChild(li);
+  li = listItem('dish', 'norimaki-sushi', 'Norimaki sushi');
+  li.childNodes[0].classList.add('dish-description');
+  img = document.createElement('img');
+  img.src = '../dist/assets/images/norimaki.jpg';
+  img.alt = 'Norimaki sushi';
+  li.appendChild(img);
+  ul.appendChild(li);
+  li = listItem('dish', 'oshizushi-sushi', 'Oshizushi sushi');
+  li.childNodes[0].classList.add('dish-description');
+  img = document.createElement('img');
+  img.src = '../dist/assets/images/oshizushi.jpg';
+  img.alt = 'Oshizushi sushi';
+  li.appendChild(img);
+  ul.appendChild(li);
+  li = listItem('dish', 'uramaki-sushi', 'Uramaki sushi');
+  li.childNodes[0].classList.add('dish-description');
+  img = document.createElement('img');
+  img.src = '../dist/assets/images/uramaki.jpg';
+  img.alt = 'Uramaki sushi';
+  li.appendChild(img);
+  ul.appendChild(li);
+  li = listItem('dish', 'temaki-sushi', 'Temaki sushi');
+  li.childNodes[0].classList.add('dish-description');
+  img = document.createElement('img');
+  img.src = '../dist/assets/images/temaki.jpg';
+  img.alt = 'Temaki sushi';
+  li.appendChild(img);
```

Comment:
This should be on a loop, define your menu items properties on an array an build each item on a loop.  You could even map on the array to build an array of `li`s and the use append the add the resulting array with destructuring: `ul.append(...menuItems.map(`

Reply:
This is a great idea, it can reduce many lines of code.